src/java.base/share/classes/java/lang/ThreadGroup.java

```diff
@@ -185,7 +185,14 @@ public final int getMaxPriority() {
      * @return  {@code true} if this thread group is a daemon thread group;
      *          {@code false} otherwise.
      * @since   1.0
+     *
+     * @deprecated Destroying a thread group and its subgroups is not an atomic
+     *             operation and is inherently racy with respect to threads that
+     *             have been created but not have started. The concept of daemon
```

Comment:
"a daemon group" sounds better to me.

Reply:
The existing spec uses "daemon thread group" so I'd prefer to stick with that.